app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -64,6 +64,25 @@ def add_proofing_component
           update(document_check: Idp::Constants::Vendors::USPS)
       end
 
+      def localized_locations(locations)
+        return nil if locations.nil?
+        locations.map do |location|
+          {
+            address: location[:address],
+            city: location[:city],
+            distance: location[:distance],
+            name: location[:name],
+            saturday_hours: EnrollmentHelper.localized_hours(location[:saturday_hours]),
+            state: location[:state],
+            sunday_hours: EnrollmentHelper.localized_hours(location[:sunday_hours]),
+            weekday_hours: EnrollmentHelper.localized_hours(location[:weekday_hours]),
+            zip_code_4: location[:zip_code_4],
+            zip_code_5: location[:zip_code_5],
+            is_pilot: location[:is_pilot],
+          }
```

Comment:
what about an approach like:
```suggestion
          location.slice(
            :address,
            :city,
            :distance,
            :name,
            :zip_code_4,
            :zip_code_5,
            :is_pilot,
          ).merge(
            saturday_hours: EnrollmentHelper.localized_hours(location[:saturday_hours]),
            sunday_hours: EnrollmentHelper.localized_hours(location[:sunday_hours]),
            weekday_hours: EnrollmentHelper.localized_hours(location[:weekday_hours]),
          )
```